bootstrap-salt.sh

```diff
@@ -1222,6 +1222,42 @@ __gather_bsd_system_info() {
 
 
 #---  FUNCTION  -------------------------------------------------------------------------------------------------------
+#          NAME:  __gather_darwin_system_info
+#   DESCRIPTION:  Discover Darwin (aka OSX aka macOS) systems information
+#----------------------------------------------------------------------------------------------------------------------
+__gather_darwin_system_info() {
+    DISTRO_NAME=${OS_NAME}
+    DISTRO_VERSION=$(sw_vers | grep ProductVersion |  cut -f 2 -d: | tr -d ' \t')
+
+    DISTRO_MAJOR_VERSION=$(echo ${DISTRO_VERSION} | cut -d. -f 1)
+    DISTRO_MINOR_VERSION=$(echo ${DISTRO_VERSION} | cut -d. -f 2)
```

Comment:
Lint checker fails on the two lines above: remember to quote variable and command substitutions.